recipes/yaml-cpp/all/patches/0001-install-0.7.0.patch

```diff
@@ -0,0 +1,17 @@
+# see issue https://github.com/jbeder/yaml-cpp/pull/847
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b230b9e..35dfdca 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,9 +24,8 @@ option(YAML_BUILD_SHARED_LIBS "Build yaml-cpp shared library" ${BUILD_SHARED_LIB
+ cmake_dependent_option(YAML_CPP_BUILD_TESTS
+   "Enable yaml-cpp tests" ON
+   "BUILD_TESTING;CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
+-cmake_dependent_option(YAML_CPP_INSTALL
+-  "Enable generation of yaml-cpp install targets" ON
+-  "CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
++option(YAML_CPP_INSTALL
++  "Enable generation of yaml-cpp install targets" ON)
```

Comment:
With this change (or something in the same spirit), this patch could be upstreamed:
```suggestion
@@ -24,9 +24,12 @@ option(YAML_BUILD_SHARED_LIBS "Build yaml-cpp shared library" ${BUILD_SHARED_LIB
 cmake_dependent_option(YAML_CPP_BUILD_TESTS
   "Enable yaml-cpp tests" ON
   "BUILD_TESTING;CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
-cmake_dependent_option(YAML_CPP_INSTALL
-  "Enable generation of yaml-cpp install targets" ON
-  "CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
+set(YAML_MAIN_PROJECT OFF)
+if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
+  set(YAML_MAIN_PROJECT ON)
+endif()
+option(YAML_CPP_INSTALL
+  "Enable generation of yaml-cpp install targets" ${YAML_MAIN_PROJECT})
```

Reply:
It is already reported but not merged yet - see patch header
https://github.com/conan-io/conan-center-index/pull/6268/files/14615cf8fefee13d5d6350e3bb8765a45ab6f42a#diff-552386d9e4a343fcd90d5921ad7d29ff59efa772289097df48da091880df7b93R1

Reply:
Oops. Thanks!